### PR TITLE
Update the docs for the license script

### PR DESCRIPTION
**Pull request by jason-simmons**

### CODE REVIEW

**Comment by jason-simmons** [comment]
@Hixie

File: tools/licenses/README [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
 pub get
 gclient sync
-dart lib/main.dart ../../.. > ../../sky/packages/sky_engine/LICENSE
-dart --checked lib/main.dart ../../.. > ../../travis/licenses.golden
+dart lib/main.dart --src ../../.. --out ../../../out/licenses --golden ../../travis/licenses_golden
+```
+
+Then copy any affected files from `../../../out/licenses` to `../../travis/licenses_golden` and add them to your change.
+
+To update the license file used for product releases, run:
```
Comment (goderbauer) [inline tools/licenses/README]: Can you add a comment when the license for product release has to be updated?

Comment (Hixie) [inline tools/licenses/README]: It should be updated any time the golden file changes in a way that involves changes to anything other than the FILE lines. In other words, whenever the actual licenses that apply to each package change.

Comment (jason-simmons) [inline tools/licenses/README]: Updated the readme with @Hixie's description

**Comment by Hixie** [comment]
LGTM

**Pull request merged**